lib/version.js

```diff
@@ -22,7 +23,13 @@ class NodeVersion {
 	}
 
 	static get defaultArch() {
-		return NodeVersion._standardArchName(process.arch);
+		let arch = process.arch
+		if (arch === 'ppc64') {
```

Comment:
I think this code needs to go inside `_standardArchName()`, similar to how that function already handles the arm version. That way nvs will also use 'ppc64le' as appropriate when parsing a version specified on the command-line.

Reply:
updated PTAL